### ISSUE TITLE
Instructions to use Arduino IDE on Mac OS X

### ISSUE THREAD

**Issue by fcohen** [Xinyuan-LilyGO/TTGO_TWatch_Library#50]
Are there instructions for developing on Arduino IDE on Mac OS X (Catalina)? I installed the TTGO_TWatch_Library and see under File -> Examples the TTGO examples are listed as INCOMPATIBLE. Also, I'm not sure what I should be doing to compile for ESP32? -Frank

**Comment by Altron-2k20** [comment]
Have you installed the additional board definitions ?
Menu Arduino -> Preferences -> Additional Boards Manager -> Insert the following URL: https://dl.espressif.com/dl/package_esp32_index.json

**Comment by fcohen** [comment]
Thank you, I got it to work! :-) Arduino IDE complains of INCOMPATIBLE examples until I 1) Installed the ESP32 dev tools, 2) Installed the ESP32 board definitions, 3) Updated the ESP32 board location in the Arduino IDE Preferences. And now I changed config.h for the Batman example, chose the TTGO T1 board from the Tools -> Board menu in IDE, selected the correct USB port, and clicked Upload. Thanks for your support.
